specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachineScaleSet.json

```diff
@@ -3949,6 +3962,13 @@
             "lun"
           ],
           "description": "The data disks."
+        },
+        "diskControllerType": {
+          "type": "string",
+          "items": {
+            "$ref": "./computeRPCommon.json#/definitions/DiskControllerType",
+            "description": "Specifies the disk controller type configured for the virtual machines in the scale set. This property is only supported for virtual machines whose operating system disk and and VM size supports hyperVGeneration: V2. For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes. <br>NOTE: You need to deallocate the virtual machines in the scale set before updating its disk controller type based on the upgrade mode configured for the scale set.<br><br> Minimum api-version: 2020-08-01"
```

Comment:
remove the below section as it is covered in the definition of DiskControllerType

This property is only supported for virtual machines whose operating system disk and and VM size supports hyperVGeneration: V2. For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes.

Also update the Minimum api-version to 2022-08-01

Reply:
acknowledging